packages/system/data_stream/diskio/fields/fields.yml

```
@@ -48,75 +48,4 @@
       type: long
       metric_type: counter
       description: |
-        The total number of of milliseconds spent doing I/Os.
-    - name: iostat.read.request.merges_per_sec
-      type: float
-      metric_type: gauge
-      description: |
-        The number of read requests merged per second that were queued to the device.
-    - name: iostat.write.request.merges_per_sec
-      type: float
-      metric_type: gauge
-      description: |
-        The number of write requests merged per second that were queued to the device.
-    - name: iostat.read.request.per_sec
-      type: float
-      metric_type: gauge
-      description: |
-        The number of read requests that were issued to the device per second
-    - name: iostat.write.request.per_sec
-      type: float
-      metric_type: gauge
-      description: |
-        The number of write requests that were issued to the device per second
-    - name: iostat.read.per_sec.bytes
-      type: float
-      format: bytes
-      metric_type: gauge
-      description: |
-        The number of Bytes read from the device per second.
-    - name: iostat.read.await
-      type: float
-      metric_type: gauge
-      description: |
-        The average time spent for read requests issued to the device to be served.
-    - name: iostat.write.per_sec.bytes
-      type: float
-      format: bytes
-      metric_type: gauge
-      description: |
-        The number of Bytes write from the device per second.
-    - name: iostat.write.await
-      type: float
-      metric_type: gauge
-      description: |
-        The average time spent for write requests issued to the device to be served.
-    - name: iostat.request.avg_size
-      type: float
-      format: bytes
-      unit: byte
-      metric_type: gauge
-      description: |
-        The average size (in bytes) of the requests that were issued to the device.
-    - name: iostat.queue.avg_size
-      type: float
-      unit: byte
-      metric_type: gauge
-      description: |
-        The average queue length of the requests that were issued to the device.
-    - name: iostat.await
-      type: float
-      metric_type: gauge
-      description: |
-        The average time spent for requests issued to the device to be served.
-    - name: iostat.service_time
-      type: float
-      unit: ms
-      metric_type: gauge
-      description: |
-        The average service time (in milliseconds) for I/O requests that were issued to the device.
-    - name: iostat.busy
-      type: float
-      metric_type: gauge
-      description: |
-        Percentage of CPU time during which I/O requests were issued to the device (bandwidth utilization for the device). Device saturation occurs when this value is close to 100%.
+        The total number of of milliseconds spent doing I/Os.
```

```suggestion
        Total time spent in doing I/Os in milliseconds.
```

I see there are multiple definitions where "total number of milliseconds" (e.g., total number of a unit of time) is used which is incorrect. If there are only a few errors like this perhaps we can take it up in this PR. If not, create another PR with the fixes.

---

I'd prefer not to make changes to any other field in this PR to keep things clean.
Maybe we should create a backlog issue for these errors. Can you do that?

---

#7950